config/ignition_edifice_debian_buster.yaml

```diff
@@ -2,119 +2,148 @@ name: ignition_edifice_debian_buster
 method: http://packages.osrfoundation.org/gazebo/debian-stable
 suites: [buster]
 component: main
-architectures: [amd64, source]
+architectures: [amd64, armhf, arm64, source]
```

Comment:
I would like to just standardize on pulling these architectures. Although I could be convinced to give up hope for armhf ever coming back.

Reply:
From time to time Jose tries to fix them, but it's not a priority for us, so I'd be ok removing them if they cause headaches.